ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -333,7 +326,7 @@ open class SideTabBar: UIView {
         return itemView(with: item, in: .bottom)
     }
 
-    private class func createStackView(spacing: CGFloat) -> UIStackView {
+    private func createStackView(spacing: CGFloat) -> UIStackView {
```

Comment:
I feel like that should work, but it just would require an explicit callout of which class is being invoked?

```swift
    private lazy var bottomStackView: UIStackView = {
        return SideTabBar.createStackView(spacing: tokens.bottomTabBarItemSpacing)
    }()
```

Or is that not allowed because the method is private?